<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAdd</class>
 <widget class="QDialog" name="dlgAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1170</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QWidget" name="all_2" native="true">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QWidget" name="downall" native="true">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QWidget" name="up" native="true">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <layout class="QGridLayout" name="gridLayout_9"/>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QWidget" name="down" native="true">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <widget class="QTableWidget" name="tableWidget">
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <attribute name="horizontalHeaderVisible">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="horizontalHeaderMinimumSectionSize">
                    <number>20</number>
                   </attribute>
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>140</number>
                   </attribute>
                   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderDefaultSectionSize">
                    <number>60</number>
                   </attribute>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="LeSearch">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="placeholderText">
                    <string>请根据关键字进行搜索</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="mid" native="true">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0" colspan="2">
                  <widget class="QComboBox" name="choseCBox">
                   <property name="minimumSize">
                    <size>
                     <width>160</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <item>
                    <property name="text">
                     <string>会员客户</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>衣物种类</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>员工账号</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="8">
                  <widget class="QPushButton" name="BtnreFresh">
                   <property name="minimumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>刷新</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="6">
                  <widget class="QPushButton" name="BtnDelete">
                   <property name="minimumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>删除</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="4" colspan="5">
                  <widget class="QGroupBox" name="groupBox">
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="title">
                    <string>选择衣服其他属性</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="4" column="1">
                     <widget class="QRadioButton" name="ClothesTreatment">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>特殊处理</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="0">
                     <widget class="QComboBox" name="CBWashWay">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <item>
                       <property name="text">
                        <string>全部</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>干洗</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>水洗</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>皮衣</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>单烫</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>其他</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QRadioButton" name="ClothesNameAndPrice">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>衣服名称</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QRadioButton" name="ClothesColor">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>衣服颜色</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QRadioButton" name="ClothesDefect">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>瑕疵</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="2">
                     <widget class="QRadioButton" name="ClothesEffect">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>洗后效果</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QRadioButton" name="ClothesBrand">
                      <property name="minimumSize">
                       <size>
                        <width>220</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>品牌</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="4">
                  <widget class="QLabel" name="LbCount">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>总人数：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="7">
                  <widget class="QPushButton" name="BetnUpdate">
                   <property name="minimumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>修改</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <widget class="QPushButton" name="BtnAdd">
                   <property name="minimumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>155</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>增加</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>choseCBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
